packages/usa-checkbox/src/usa-checkbox.stories.js

```diff
@@ -5,6 +5,11 @@ import TestComponent from "./test/test-patterns/test-usa-checkbox.twig";
 export default {
   title: "Components/Form Inputs/Checkbox",
   argTypes: {
+    disabled: {
+      control: {type: 'boolean'},
+      defaultValue: false,
+      name: "disabled"
+    },
```

Comment:
What do you think about using a radio controls in these stories like we do for Date picker? It makes it easier to isolate the test.

```suggestion
    disabled_state: {
      name: "Disabled state",
      control: { type: "radio" },
      options: ["none", "disabled", "aria-disabled"]
    },
```